vllm/core/block_manager.py

```diff
@@ -508,3 +523,19 @@ def get_num_cached_tokens(self, seq: Sequence) -> int:
         cached in the block manager for the sequence.
         """
         return self._computed_blocks_tracker.get_num_cached_tokens(seq)
+
+    def get_and_reset_swaps(self,
```

Comment:
This function seems like not get the real physical block ID from `get_physical_block_id`? Especially for CPU PrefixCachingBlockAllocator, whose start ID is not zero.

Reply:
If I understand correctly, this function should not return the physical block IDs because the `get_physical_block_id` will be called in `block_manager.swap_in()`/`block_manager.swap_out()` later. 

Call chain is: `scheduler._swap_in() --> block_manager.swap_in() --> block_allocator.get_physical_block_id()` (similar for swapping out).

(Let me know if my understanding is incorrect and I will fix it asap, thanks!)